CRM/Member/Form/MembershipRenewal.php

```diff
@@ -799,25 +799,12 @@ public function processMembership($contactID, $membershipTypeID, $is_test, $chan
         $numRenewTerms
       );
 
-      $currentMembership['join_date'] = CRM_Utils_Date::customFormat($currentMembership['join_date'], $format);
```

Comment:
not required because Membership::create now doesn't need existing params loaded & set